setup.py

```diff
@@ -80,9 +80,12 @@ def read(*names, **kwargs):
         "dill",
         "numpy",
         "pandas",
-        "pyomo >= 5.7.0, < 5.7.3",
+        "pyomo >= 5.7.0, < 7.0",
```

Comment:
```suggestion
        "pyomo >= 6.0.0, < 7.0",
```

The changes require `pyomo>=6`, when using 5.7, the following error occurs:

> ImportError: cannot import name 'ScalarBlock' from 'pyomo.core.base.block'